composer.json

```diff
@@ -26,7 +26,7 @@
         "symfony/form": "~2.1,!=2.5.0",
         "symfony/validator": "~2.1,!=2.5.0",
         "symfony/translation": "~2.1,!=2.5.0",
-        "doctrine/orm": "~2.2,>=2.2.3"
+        "doctrine/orm": "~2.2,>=2.2.3,<2.5"
```

Comment:
I don't think that it's a good idea. We can just leave `2.4.x` installed in `composer.lock`, but without restrictions here